src/g_canvas.h

```diff
@@ -630,6 +630,7 @@ EXTERN void guiconnect_notarget(t_guiconnect *x, double timedelay);
 /* ------------- IEMGUI routines used in other g_ files ---------------- */
 EXTERN t_symbol *iemgui_raute2dollar(t_symbol *s);
 EXTERN t_symbol *iemgui_dollar2raute(t_symbol *s);
+EXTERN t_symbol *iemgui_put_in_braces(t_symbol *s);
```

Comment:
no, please don't export this new function.
as @Ant1r pointed out in https://github.com/pure-data/pure-data/pull/953#issuecomment-616194527 there's already trouble that the `raute2dollar` and `dollar2raute`  functions are exported. (they should be deprecated and retired asap)